docs/rules/dynamic-import-chunkname.md

```diff
@@ -11,6 +11,15 @@ This rule enforces naming of webpack chunks in dynamic imports. When you don't e
 This rule runs against `import()` by default, but can be configured to also run against an alternative dynamic-import function, e.g. 'dynamicImport.'
 You can also configure the regex format you'd like to accept for the webpackChunkName - for example, if we don't want the number 6 to show up in our chunk names:
 
+<!-- begin auto-generated rule options list -->
+
+| Name                     | Type     |
+| :----------------------- | :------- |
+| `importFunctions`        | String[] |
+| `webpackChunknameFormat` | String   |
```

Comment:
`string` should be lowercase, i think

Reply:
Do you know of any related example / precedent for displaying variable/data types one way or the other, or have any other reasoning behind your preference? I'm open to considering a change.

The current reasoning is that arbitrary, user-defined values like the name of an option are displayed in backticks, whereas values from a fixed set like variable types are simply capitalized.

Reply:
Here's a comprehensive example that shows how we format different kinds of values: https://github.com/bmish/eslint-doc-generator/blob/main/docs/examples/eslint-plugin-test/docs/rules/no-foo.md#options

Reply:
`String` means an object - the constructor - and `string` means a primitive. It's what `typeof` produces for primitives as well.

TypeScript follows this convention as well.